package-examples/subpackages/wordpress/Kptfile

```diff
@@ -1,39 +1,10 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
```

Comment:
can you move this one level up and delete the `subpackages` directory?

Reply:
Done.